tasks/task06/Decorators.py

```diff
@@ -0,0 +1,39 @@
+import functools
+import time
+
+
+def measure_elapsed_time(func):
+    """Print the runtime of the decorated function"""
+    @functools.wraps(func)
+    def wrapper_timer(*args, **kwargs):
+        start_time = time.perf_counter()    # 1
+        print(f"calling {func.__name__}")
+        value = func(*args, **kwargs)
+        end_time = time.perf_counter()      # 2
+        run_time = end_time - start_time    # 3
+        print(f"{func.__name__} call took {run_time:.4f} seconds")
```

Comment:
```suggestion
        print(f"{func.__name__} call took {run_time:.1f} seconds")
```

Что бы как в примере :)